notebooks/azureml_model_headers_demo/german_credit_azure_ml_certifai_pro_demo.ipynb

```diff
@@ -0,0 +1,1130 @@
+    "4. Deploy models (predict) as web service"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Configure and Initialize Azure workspace\n",
+    "\n",
+    "- Follow the instructions listed here [creating and managing azure-ml workspace](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-manage-workspace) to create an azure-ml workspace\n",
+    "\n",
+    "**Once you have the workspace created easiest way to run through remaining steps is to download the `config.json` to the current directory and replace the exisiting config.json**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a [Workspace](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.workspace%28class%29?view=azure-ml-py) object from the persisted configuration."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core import Workspace\n",
+    "ws = Workspace.from_config()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Register models to created  workspace\n",
+    "\n",
+    "Register a file or folder as a model by calling [Model.register()](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.model.model?view=azure-ml-py#register-workspace--model-path--model-name--tags-none--properties-none--description-none--datasets-none--model-framework-none--model-framework-version-none--child-paths-none-).\n",
+    "\n",
+    "In addition to the content of the model file itself (model + scaler object), our registered model will also store model metadata like model description, tags, etc. -- that will be useful when managing and deploying models in our workspace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core.model import Model\n",
+    "\n",
+    "logistic_model_azure = Model.register(model_path=logistic_model_disk_path,\n",
+    "                       model_name=logistic_model_name,\n",
+    "                       tags={'area': \"banking credit risk\", 'type': \"classification\"},\n",
+    "                       description=\"Logistic Classifier model to predict credit loan approval\",\n",
+    "                       workspace=ws)\n",
+    "\n",
+    "svm_model_azure = Model.register(model_path=svm_model_disk_path,\n",
+    "                       model_name=svm_model_name,\n",
+    "                       tags={'area': \"banking credit risk\", 'type': \"classification\"},\n",
+    "                       description=\"Support Vector Machine Classifier model to predict credit loan approval\",\n",
+    "                       workspace=ws)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a custom prediction environment inside azure-ml workspace\n",
```

Comment:
To keep this simpler and more focused on Pro, can we drop creating the custom env and just use the default one? Also reduce test cases.

Reply:
I can change this so that the following is added to pre-requisites for this notebook:

1. Service Keys for the example German Credit Model hosted on Azure ML as described in the other notebook and link them to that notebook.
2. User will only enter the Service Keys of the model here and all cells pertaining to model training and deployment can be dropped out of this notebook

Reply:
Good idea.